api/envoy/extensions/compression/zstd/compressor/v3/zstd.proto

```diff
@@ -34,25 +34,29 @@ message Zstd {
     BTULTRA2 = 9;
   }
 
-  // Set compression parameters according to pre-defined cLevel table.
+  // Set compression parameters according to pre-defined compression level table.
   // Note that exact compression parameters are dynamically determined,
-  // depending on both compression level and srcSize (when known).
+  // depending on both compression level and source content size (when known).
   // Value 0 means default, and default level is 3.
   // Setting a level does not automatically set all other compression parameters
   // to default. Setting this will however eventually dynamically impact the compression
   // parameters which have not been manually set. The manually set
   // ones will 'stick'.
   google.protobuf.UInt32Value compression_level = 1;
 
-  // A 32-bits checksum of content is written at end of frame (default:0)
+  // A 32-bits checksum of content is written at end of frame, If not set, defaults to false.
   bool enable_checksum = 2;
 
   // The higher the value of selected strategy, the more complex it is,
   // resulting in stronger and slower compression.
   // Special: value 0 means "use default strategy".
   Strategy strategy = 3 [(validate.rules).enum = {defined_only: true}];
 
-  // Dictionary for compression.
+  // A dictionary for compression. zstd offers dictionary compression, which greatly improves
```

Comment:
im guessing that zstd should be upper-case (im guessing its an acronym) - either way i think this should at least be capitalized at the beginning of a sentence

same elsewhere re upper-case

Reply:
done